stack.yaml.lock

```diff
@@ -32,10 +32,10 @@ packages:
   original:
     hackage: hex-text-0.1.0.0
 - completed:
-    hackage: named-0.3.0.1@sha256:2975d50c9c5d88095026ffc1303d2d9be52e5f588a8f8bcb7003a04b79f10a06,2312
+    hackage: named-0.3.0.1@sha256:418543272f0c844a77639b05276ee3906d5f305cd4cab46fafcb4f15b7f1e60d,2312
```

Comment:
It's quite odd that you had to update this, hopefully that's because you are based on an older version of my PR.

Since that's been merged a rebase should remove this need, I think.